tests/cypress/e2e/unit_tests/p1_fleet.spec.ts

```diff
@@ -316,3 +316,71 @@ if (!/\/2\.7/.test(Cypress.env('rancher_version'))) {
     )  
   });
 
+  describe('Test Self-Healing of IMMUTABLE resources modification when correctDrift option used for exisiting GitRepo', { tags: '@p1'}, () => {
+    const correctDriftTestData: testData[] = [
+      { qase_id: 80,
+        repoName: "local-cluster-correct-80",
+        resourceType: "ConfigMaps",
+        resourceName: "mp-app-config",
+        resourceNamespace: "test-fleet-mp-config",
```

Comment:
By adding here extra line: `dataToAssert: "test, test_key"` major refactor can be done later

Reply:
Done